src/fastmcp/utilities/logging.py

```diff
@@ -16,7 +16,7 @@ def get_logger(name: str) -> logging.Logger:
     Returns:
         a configured logger instance
     """
-    return logging.getLogger(f"FastMCP.{name}")
+    return logging.getLogger(f"fastmcp.{name}")
```

Comment:
Is there a reason to do this vs just doing getLogger(__name__) or whatever in each file?